tequilapi/client/client.go

```diff
@@ -106,16 +106,28 @@ func (client *Client) Disconnect() (err error) {
 	return nil
 }
 
+// ConnectionStatistics returns statistics about current connection
+func (client *Client) ConnectionStatistics() (statistics StatisticsDTO, err error) {
```

Comment:
Should not it be getter?

Reply:
It is a getter in a way, but it doesn't need `Get` prefix:
From https://golang.org/doc/effective_go.html#Getters:
> it's neither idiomatic nor necessary to put Get into the getter's name. If you have a field called owner (lower case, unexported), the getter method should be called Owner (upper case, exported), not GetOwner

Reply:
Cool.
Would be good to consistent in one object, I remember something like  `GetIP` and `GetStatus`